__tests__/sum.test.js

```diff
@@ -0,0 +1,68 @@
+const Q = require('q');
+const request = Q.denodeify(require('request'));
+
+const MockClient = require('../src/client/Client');
+
+describe('examples', () => {
+	it('simple mock', done => {
+		return request('http://localhost:2021/simple-mock-example', {json: true}).spread((response, body) => {
+			expect(response.statusCode).toBe(200);
+			expect(body).toEqual({someKey: 'some value'});
+		})
+		.then(done, done.fail);
+	});
+
+	it('dynamic mock', done => {
+		return request('http://localhost:2020/dynamic-mock-example').spread((response, body) => {
+			expect(response.statusCode).toBe(200);
+			expect(body).toEqual('Counter: 1');
+		})
+		.then(() => {
+			return request('http://localhost:2020/dynamic-mock-example').spread((response, body) => {
+				expect(response.statusCode).toBe(200);
+				expect(body).toEqual('Counter: 2');
+			});
+		})
+		.then(done, done.fail);
+	});
+});
+
+describe('clean', () => {
+	it('should remove requests and listeners', done => {
+		const mockClient = new MockClient(2020, 'localhost:3000');
+
+		// Add route
+		mockClient.addRoute({
+			uri: '/test',
+			method: 'get',
+			response: {
+				body: 'hello world'
+			}
+		})
+
```

Comment:
I'd like to get rid of all the blank line here because it's chained together to be one statement.